scss/organisms/_banners.scss

```diff
@@ -288,10 +288,12 @@
 .updated .notice-dismiss {
 	z-index: 1;
 	text-decoration: none;
-	&:before {
-		color: $white;
-	}
 }
+
+.jp-wpcom-connect__container-top-text + .jp-wpcom-connect__inner-container .notice-dismiss::before {
```

Comment:
Ha! This isn't too bad. Let's just make sure to remove it after the A/B test has run its course.